.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 
-*           @ableuler @olevski @SwissDataScienceCenter/YAT
+*           @ableuler @olevski @SwissDataScienceCenter/YAT @SwissDataScienceCenter/renku-python-maintainers
```

Comment:
remove ableuler ?

Reply:
I added Andreas with write access as an external collaborator. If he accepts the invite this will clear the error.